pkg/adapter/apiserver/adapter_test.go

```diff
@@ -32,6 +32,12 @@ import (
 	rectesting "knative.dev/eventing/pkg/reconciler/testing"
 )
 
+type mockReporter struct{}
+
+func (r *mockReporter) ReportEventCount(args *ReportArgs, responseCode int) error {
+	return nil
```

Comment:
I would still prefer if we asserted the metrics were incremented in the test, but I won't hold up this PR for it.